lib/structures/Message.js

```diff
@@ -392,6 +392,33 @@ class Message extends Base {
         return this._client.unpinMessage.call(this._client, this.channel.id, this.id);
     }
 
+    /**
+    * Delete the message as a webhook
+    * @param {String} token The token of the webhook
+    * @returns {Promise}
+    */
+    webhookDelete(token) {
```

Comment:
Nitpick, but imo it would be better to merge this into the `delete` method. A little bit of ternary magic seems nicer.

Reply:
Laziness :-)
I'll ask abal about this and see if he wants it in a single method

Reply:
User needs to check if a token should be provided anyway, better to separate functions separate